flow/project.py

```diff
@@ -541,6 +539,7 @@ def _setup_template_environment(self):
         template_environment = jinja2.Environment(
             loader=jinja2.ChoiceLoader(load_envs),
             trim_blocks=True,
+            lstrip_blocks=True,
```

Comment:
I found that adding this could allow jinja2 to ignore indent in all logic codes. I think this could work for the status print template but I want to check with you @csadorf if that won't affect the job submission script as this change is global.

Reply:
I'd prefer to not the change the global template context to avoid breaking backwards compatibility.

Reply:
I removed this line